Cargo.toml

```diff
@@ -51,6 +51,7 @@ url = "2"
 # https://crates.io/crates/cargo-deb
 [package.metadata.deb]
 depends = ""
+section = "utility"
```

Comment:
You list specifies it as `utils`. But I think `net` might also be a good fit?

Reply:
`net` indeeds carries a better meaning: "Daemons and clients to connect your system to the world." ([source](https://packages.debian.org/unstable/)). Thanks for the review.